tests/e2e/sriovoperatornodepolicy_test.go

```diff
@@ -193,9 +189,7 @@ var _ = Describe("Operator", func() {
 				Mtu:      9000,
 				NumVfs:   6,
 				NicSelector: sriovnetworkv1.SriovNetworkNicSelector{
-					PfNames:     []string{"ens803f1#0-5"},
-					Vendor:      "8086",
-					RootDevices: []string{"0000:86:00.1"},
+					PfNames:     []string{"#0-5"},
```

Comment:
This compromises a bit the readability, because it's not clear it's going to appended later on.
How about having a placeholder that you replace with the real pfname? something like PFNAME#0-5 and then you replace it later on?